source/manual/on-call.html.md

```diff
@@ -23,6 +23,24 @@ to keep GOV.UK running at night, on the weekends and on public holidays.
 - Logs are not as important as being available - if you need to lose some logs
   in order to bring the site back up, that's probably a good trade-off to make
 
+## On call checklist
+
+You should do these things before going on call so you're prepared.
+
+1. Have the numbers of other people on your shift saved in your phone. This
+   includes whoever is on Escalations. Get these numbers from PagerDuty.
+2. Ensure you have an up to date local copy of the [Developer Docs][docs] repository
+   and that you can build it.
+3. Make sure your [`fabric-scripts`][fabric] are up to date.
+4. Make sure you can VPN to the office or disaster recovery location
```

Comment:
I have a few more things in my list to check I can access:

- decrypting 2nd line pass and encrypted hieradata (upgrades have been known to break my gpg setup...)
- ssh to mirror boxes (this may not be that important any more? depends on whether we now have other ways of handling the various times when we might need to urgently update content on the mirrors)
- production vCloud Director

This covers the things I would usually use less frequently, on the assumption that I would have noticed already if I couldn't get to production machines or log into jenkins.

Reply:
Thanks, added.